code/ai/ai_profiles.cpp

```diff
@@ -525,7 +525,9 @@ void parse_ai_profiles_tbl(const char *filename)
 
 				set_flag(profile, "$respect ship axial turnrate differences:", AI::Profile_Flags::Use_axial_turnrate_differences);
 
-				set_flag(profile, "$ships with no shields can manage ETS:", AI::Profile_Flags::nonshielded_ships_can_manage_ets);
+				set_flag(profile, "$ships with no shields can manage ETS:", AI::Profile_Flags::all_nonshielded_ships_can_manage_ets);
```

Comment:
I would make the text explicit, such as `$any ship with no shields can manage ETS:`.  Keep the old string too, for compatibility, but add a deprecation warning.

Reply:
Good idea, committed!